src/engine/SCons/Defaults.xml

```diff
@@ -572,15 +572,21 @@ searching the repositories.
 
 <scons_function name="DefaultEnvironment">
 <arguments signature="global">
-([args])
+([**kwargs])
 </arguments>
 <summary>
 <para>
-Creates and returns a default construction environment object.
-This construction environment is used internally by SCons
-in order to execute many of the global functions in this list,
-and to fetch source files transparently
+Creates and returns a default &consenv; object.
```

Comment:
maybe change to "the default" instead of "a default"?

Reply:
Sure.  Meanwhile... at what point does this instantiate by itself if you don't call this early?  I mean, is there a sequence of events that would cause it already to be instantiated, and so ignore your passing of args... if that can happen, does it need mention?